zcash_client_sqlite/src/lib.rs

```diff
@@ -703,12 +696,12 @@ impl<P: consensus::Parameters> WalletWrite for WalletDb<rusqlite::Connection, P>
                     TransferType::Incoming => {
                         match spending_account_id {
                             Some(id) => {
-                                if id != output.account {
+                                if id != *output.account() {
                                     panic!("Unable to determine a unique account identifier for z->t spend.");
                                 }
```

Comment:
We need to figure out why we're enforcing this "one account only for outputs" rule but aren't doing anything with the account ID that we resolve on.

Reply:
Opened https://github.com/zcash/librustzcash/issues/1238